bigbluebutton-html5/imports/ui/components/shortcut/component.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+export default class Shortcut extends Component {
+  constructor() {
+    super();
+
+    this.handleShortcut = this.handleShortcut.bind(this);
+  }
+
+  componentDidMount() {
+    document.addEventListener('keydown', this.handleShortcut, false);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.handleShortcut, false);
+  }
+
+  handleShortcut(event) {
+    const { shortcut } = this.props;
```

Comment:
`<Shortcut shortcut />`?

Probably should rename this prop to `combo`, `keys`, `pattern` or something like this.